design-patterns/gc-intro/index.json-ld

```diff
@@ -52,9 +52,14 @@
 			}
 		]
 	},
-	"dependencies": [
-		"bgimg"
-	],
+	"dependencies": {
+		"en": [
+			"Background image"
+		],
+		"fr": [
+			"Image d'arrière plan"
+		]
+	},
```

Comment:
Can we adjust in a way to allow us to include a link and keep the technical name?

Like:

```suggestion
	"dependencies": {
		"en": [
			{
				"title": "Background image",
				"link": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/bgimg/bgimg-en.html",
				"component": "bgimg"
			}
		],
		"fr": [
			{
				"title": "Image d'arrière plan",
				"link": "https://wet-boew.github.io/gcweb-compiled-demos/wetboew-demos/bgimg/bgimg-fr.html",
				"component": "bgimg"
			}
		]
	},
```